vulcan.yaml

```diff
@@ -25,6 +25,29 @@ conf:
   # Number of checks to run concurrently
   concurrency: 5
 
+policies:
```

Comment:
Could you update README.md with usage info?

Reply:
I added a section for policies in the README, it should be completed once we release a initial set of policies.